public/product/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>もくプロ | 制作物一覧</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.min.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.2/css/all.css" integrity="sha384-vSIIfh2YWi9wW0r9iZe7RJPrKwp6bG+s9QZMoITbCckVJqGCCRhc+ccxNcdpHuYu" crossorigin="anonymous">
+  <link rel='stylesheet' href="./style.css">
+</head>
+<body>
+  <header>
+    <div class="container">
+      <nav class="navbar" role="navigation" aria-label="main navigation">
+        <div class="navbar-brand">
+          <a class="navbar-item" href="#">
+            <h1 class="title">もくプロ</h1>
+          </a>
+      
+          <!-- ハンバガーメニュー-->
+          <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false" data-target="navMenu">
+            <span aria-hidden="true"></span>
+            <span aria-hidden="true"></span>
+            <span aria-hidden="true"></span>
+          </a>
+        </div>
+      
+        <div id="navMenu" class="navbar-menu">
+          <div class="navbar-end">
+            <a class="navbar-item" href="#">
+              もくプロについて
+            </a>
+      
+            <a class="navbar-item" href="#">
+              メンバー
+            </a>
+
+            <a class="navbar-item" href="#">
+              制作物
+            </a>
+
+            <a class="navbar-item" href="#">
+              お問い合わせ
+            </a>
+
+            <div class="navbar-item">
+              <div class="field is-grouped">
+                <p class="control">
+                  <a class="button is-info">
+                    <span class="icon">
+                      <i class="fab fa-twitter" aria-hidden="true"></i>
+                    </span>
+                  </a>
+                </p>
+                <p class="control">
+                  <a class="button is-info">
+                    <span class="icon">
+                      <i class="fab fa-instagram" aria-hidden="true"></i>
+                    </span>
+                  </a>
+                </p>
+              </div>
+            </div>
+      
+          </div>
+        </div>
+      </nav>
+    </div>
+  </header>
+
+  <main>
+    <section class="hero is-primary is-bold">
+      <div class="hero-body">
+          <p class="title">
+            制作物一覧
+          </p>
+          <p class="subtitle">
+            もくプロ
+          </p>
+      </div>
+    </section>
+    <section class='section'>
+      <div class='content has-text-centered'>
```

Comment:
細かいけど、クォーテーションの種類は全部統一したほうが美しい
```suggestion
    <section class="section">
      <div class="content has-text-centered">
```